examples/gcp-cloudrun-postgres/generate_keys.sh

```diff
@@ -13,7 +13,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-cd ../opr-core
+cd ../../opr-core
+npm i
```

Comment:
I think it might be slightly better to remove this line and change the next line to `npx run` instead of `npm run`. I believe `npx run` will install the needed libraries if necessary, but it will use libraries hoisted to a higher level if possible. That approach is a little more consistent with our use of lerna. I don't think there's any real harm in running `npm i` first, but I'd prefer not to document a use of npm install in one of these subdirectories, since that's usually the wrong thing to do (because we prefer they do their npm installs via a top-level call to `npx lerna bootstrap --hoist`

Reply:
Makes sense.  I haven't used Lerna before, looks interesting.